slurm/run-tar_make.slurm

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#SBATCH --job-name=insal_mk             # human-readable label for squeue output
+#SBATCH --time=8:00:00                  # maximum time for this job
+#SBATCH --output=insal_mk_%u.out        # user's output file (password here!)
+#SBATCH --partition=cpu                 # which SLURM partition to use
+#SBATCH --nodes=1                       # only request one node
+#SBATCH --ntasks-per-node=1             # only request one core
+#SBATCH --mail-type=ALL                 # enable email notifications for job status. Sends to submitting user
+
+module load singularity
+
+#Container path
+#argument 1 is the name of the container to use.
+CONTAINER="/caldera/projects/usgs/water/impd/pump/drb-inland-salinity/containers/${1}"
```

Comment:
So does this mean you give it an argument when you call sbatch? Like 
```
sbatch run-tar_make.slurm mycontainer.sif
```
?

If so, I wonder if a short readme in this file that specifies that should be included.

Reply:
Yes, that's right. I can add that here and in the other slurm script